File: Javascript/Algorithms/1822.js
/**
 * @param {number[]} nums
 * @return {number}
 */
let arraySign = nums => {
    let res = 1;
    for (let i of nums) {
        if (i == 0)
            return 0;
        if (i < 0)
            res = -res;
    }
    return res;
};

File: Rust/Algorithms/1822.rs
impl Solution {
    pub fn array_sign(nums: Vec<i32>) -> i32 {
        let mut res = 1;
        for i in nums {
            if i == 0 {
                res = 0;
            } 
            if i < 0 {
                res = -res;
            }
        }
        res
    }
}